submission/ContextTabs/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "manifest_version": 2,

```

Comment:
your browser extension must be using Manifest Version 3. It's fine if you can't support Firefox for now as a result of that.